### ISSUE TITLE
[2.0.0022] Checking one "public" checkbox and setting "type" of one asset is reflected for all assets.

### ISSUE THREAD

**Issue by srinivas4912** [blackberry/VSPlugin#650]
Preq: Install VS2013 plugin from v2.0.0022 on Win8.1
1. Open bar-descriptor.xml file
2. From Asset Configuration, select "Device-Debug"
3. Browse to the output directory file and locate the File that was just built & Select it
4. Repeat the step 3 for 4 times
5. Click on any one check box from Assets

Actual Result: 
1. When we check one of the check box from "Public" column, all checkboxes are being selected.
2. Similarly, When we change "Type" of one asset it is reflecting to all the other.

**Comment by dbrgss** [comment]
Cannot reproduce this issue.   When in step 4 you are repeating and adding more assets...  are you choosing different files??

**Comment by srinivas4912** [comment]
No, choosing same file from the same project and it is reproducing using VS2012 plugin from v2.0.0022 also.
Adding VS 2012 label.

**Comment by srinivas4912** [comment]
I'm able to add same asset file multiple times, is it expected behavior?

**Comment by dbrgss** [comment]
Well it's expected as there is no validation on files being added into the Assets.    That being said... it makes no sense to add the same asset more than once.   The behavior makes more sense now though... as when you click a check box... the code is going through and checking all entries with the same name.  Assigning P-Value of 3

**Comment by hkopparru** [comment]
showing the same behavior in 3.0.0001 plugin VS2013
